### DISCUSSION TITLE
2.4 Pro Tip: You should never edit any SLS files on the command line.

### DISCUSSION THREAD

**Discussion by TOoSmOotH** [Security-Onion-Solutions/securityonion#11054] [2.4]
We did an initial pass on the documentation before GA release but you will still find some references to modifying sls files. We are in the process of updating the docs but I wanted to stress that ALL modifications are made in 2.4 via the configuration section in SOC. If you start copying files to local and modifying them you will experience problems.

**Comment by CharlieMK** [comment]
Is there a workaround for this? I used to have quite some scripts modifying config inside security onion. With 2.4, these changes are indeed reverted or cause issues.

**Reply by TOoSmOotH** [reply]
Many pillar locations have changed in 2.4. If you change things in scripts keep in mind that anything done in the config interface will replace those changes if the user touches them. We have additional checks with the config interface to validate certain fields. If you have mistakes in your modifications it can cause the entire config interface not to load. Replacing the certs via scripts shouldn't be that big of a deal but you will want to test it.
